### PR TITLE
Fix the issue with anonymous view

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: apps/portal/js/designer.js [hunk 1]
```diff
@@ -445,14 +445,31 @@ $(function () {
...
+        //    pageType = ANONYMOUS_DASHBOARD_VIEW;
+        //}
+
+        //if(((checked && !ues.global.isSwitchToNewPage)|| (!checked && ues.global.isSwitchToNewPage) && ues.global.anon)){
+        //    pageType = DEFAULT_DASHBOARD_VIEW;
+        //    //pageType = ANONYMOUS_DASHBOARD_VIEW;
+        //}else {//if(ues.global.anon && (!checked && ues.global.isSwitchToNewPage)){
+        //    pageType = ANONYMOUS_DASHBOARD_VIEW;
+        //    //pageType = DEFAULT_DASHBOARD_VIEW;
+        //}
```
Comment (manuranga) [inline apps/portal/js/designer.js]: commented code should be deleted

File: apps/portal/js/designer.js [hunk 2]
```diff
@@ -950,26 +974,39 @@ $(function () {
...
         }
+
         if (anon.is(':checked')) {
             ues.global.dbType = ANONYMOUS_DASHBOARD_VIEW;
             dashboard.isanon = true;
             page.isanon = true;
             $(".toggle-design-view").removeClass("hide");
-            $('#toggle-dashboard-view').bootstrapToggle('on');
+
+            debugger;
```
Comment (manuranga) [inline apps/portal/js/designer.js]: debugger has to be deleted.

File: apps/portal/js/designer.js [hunk 3]
```diff
@@ -595,16 +613,22 @@ $(function () {
             if(checked){
                 pageType = DEFAULT_DASHBOARD_VIEW;
                 ues.global.type = DEFAULT_DASHBOARD_VIEW;
+                //ues.global.anon = false;
```
Comment (sinthuja) [inline apps/portal/js/designer.js]: Remove the commented code if it's no used.

File: apps/portal/js/designer.js (hunk 2 above)
Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: This is deleted.

File: apps/portal/js/designer.js (hunk 1 above)
Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: This has been fixed

File: apps/portal/js/designer.js (hunk 3 above)
Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: Fixed in the PR https://github.com/wso2/carbon-dashboards/pull/173

File: apps/portal/js/designer.js [hunk 4]
```diff
@@ -445,14 +445,31 @@ $(function () {
...
     var destroyPage = function (page, pageType, done) {
+        debugger;
         var checked = $('#toggle-dashboard-view').prop('checked');
         var area;
         pageType = pageType ? pageType : DEFAULT_DASHBOARD_VIEW;
-        if(((checked && !ues.global.isSwitchToNewPage) || (!checked && ues.global.isSwitchToNewPage) && ues.global.anon)){
-            pageType = ANONYMOUS_DASHBOARD_VIEW;
-        }else{
+
+        if(!page.isanon && checked){
```
Comment (Kishanthan) [inline apps/portal/js/designer.js]: code formatting issue. please fix them throughout all the places.

**Pull request merged**